tipuesearch_lite.js

```diff
@@ -135,9 +117,32 @@ window.onload = function execute(){
             }
             resultsHTML += "</div>";
         }
-    // give the page the actual contents, which were build up
-    document.getElementById("tipue_search_content").innerHTML = resultsHTML;
+
+        // add information to beginning of the output
+        resultsHTML = getSearchInfo(results, startTimer, commonTermHits) + resultsHTML;
+        // give the page the actual contents, which were build up
+        document.getElementById("tipue_search_content").innerHTML = resultsHTML;
     }
+
+    function getSearchInfo(results, startTimer, commonTermHits) {
```

Comment:
Not a fan of this function name. I don't think you're literally "getting search information". IMO you're building the HTML results output.